### PR TITLE
Build on loongarch64

### PR DESCRIPTION
**Pull request by xiangzhai**
Hi,

Passed test on loongarch64:

```
[info] Passed: Total 104, Failed 0, Errors 0, Passed 104, Ignored 1
```

Please review my patch.

Thanks,
Leslie Zhai

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -240,6 +240,9 @@ linux-s390x: jni-header
 windows-aarch64: jni-header
 	./docker/dockcross-windows-arm64 -a $(DOCKER_RUN_OPTS) bash -c 'make clean-native native CROSS_PREFIX=/usr/xcc/aarch64-w64-mingw32-cross/bin/aarch64-w64-mingw32- OS_NAME=Windows OS_ARCH=aarch64'
 
+linux-loongarch64: jni-header
+	./docker/dockcross-loongarch64 -a $(DOCKER_RUN_OPTS) bash -c 'make clean-native native CROSS_PREFIX=/usr/xcc/loongarch64-unknown-linux-gnu/bin/loongarch64-unknown-linux-gnu- OS_NAME=Linux OS_ARCH=loongarch64'
```
Comment (xerial) [inline Makefile:244]: ./docker/dockcross-loongarch64 This file is missing. Could you add it?

Comment (xiangzhai) [reply Makefile]: Hi @xerial 

Thanks for your careful review!

Sorry there is no dockcross for LoongArch64 right now.

Thanks,
Leslie Zhai

File: Makefile [hunk 2]
```diff
@@ -151,7 +151,7 @@ native: jni-header snappy-header $(NATIVE_DLL)
 native-nocmake: jni-header $(NATIVE_DLL)
 snappy: native $(TARGET)/$(snappy-jar-version).jar
 
-native-all: native native-arm clean-docker mac64 win32 win64 linux32 linux64 linux-ppc64le linux-riscv64 linux-s390x
+native-all: native native-arm clean-docker mac64 win32 win64 linux32 linux64 linux-ppc64le linux-riscv64 linux-s390x linux-loongarch64
```
Comment (xerial) [inline Makefile:154]: If no dockcross image is available, I can't accept this change as CI will be broken. Could you revert this line?

File: Makefile (hunk 1 above)
Comment (xerial) [reply Makefile]: Ok. If so I can't maintain LoongArch64 version for upcoming snappy-java versions. I'm not sure this is what you expect.

File: Makefile (hunk 2 above)
Comment (xiangzhai) [reply Makefile]: Done.

Thanks,
Leslie Zhai

**Comment by xerial** [comment]
Although I cannot guarantee the future support of loongarch64, adding these change itself for the next versionn or snapshot versions should have no harm. I'll merge them. 

Note. If you need to use loongarch64 in the future versions of snappy-java, you need to submit a PR like this again.

**Pull request merged**